app/src/main/java/mozilla/lockbox/store/DataStore.kt

```diff
@@ -158,9 +158,17 @@ open class DataStore(
     private fun reset() {
         clearList()
         backend.reset().then({
+            if (backend is MemoryLoginsStorage) {
+                updateList()
+            }
+
```

Comment:
Yeah, this is kinda weird. I think we should probably be calling a `DataStoreAction.Sync` at some point.